string-manipulation-2/src/last-chars.js

```diff
@@ -0,0 +1,16 @@
+/* exported lastChars */
+
+/*
+  type checks
+  slice from length to str.length
+    handle if length > str.length (in that case slice from 0 not a negative number)
+*/
+
+function lastChars(length, str) {
+  if (typeof length !== 'number' || length < 0 || typeof str !== 'string') return 'invalid'
+  if (length > str.length) {
+    return str.slice(0, str.length)
```

Comment:
Or, you can just return `str`

Reply:
HAHAHA I'm overcomplicating things again 😭